woocommerce/woocommerce-ios
# Update Networking, Storage and Yosemite to "recommended settings"

Closes #3020 

Upgrade the project settings, as suggested by Xcode, for:
* Networking
* Storage
* Yosemite

This should silence all the warnings due to "Upgrade to recommended settings" with one exception: the Pods target

# What's new
* Changes introduced by Xcode when letting it upgrade the mentioned subprojects to the recommended settings

# How to test
* Checkout the branch
* Build and run
* Check that there is only one warning with the text "Upgrade to recommended settings" and that said warning is emitted by the Pods project
* CI Should be ✅ as well. 

Update release notes:
